src/utils/utf8.c

```diff
@@ -70,6 +85,17 @@ utf8_ctx_create(const char *locale)
 		return NULL;
 	}
 
+	ec = U_ZERO_ERROR;
+	UChar *translit_id;
+	translit_id = malloc(sizeof(NFKD_RULE) * sizeof(UChar));
+	u_strFromUTF8(translit_id, sizeof(NFKD_RULE), NULL, NFKD_RULE, -1, &ec);
```

Comment:
While this shouldn't fail, you can add an error check just in case: `if (U_FAILURE(ec)) {` (don't forget to free translit_id in the error branch).